site/docs/guides/spark-s3.md

```diff
@@ -54,7 +54,52 @@ for the most up-to-date information on how to connect Iceberg to S3.
 Note: the word `nessie` in configuration property names is the name of the Nessie catalog in the Spark session.
 A different name can be chosen according the user's liking.
 
-Then, in `spark-sql` issue a `use` statement to make `nessie` the current catalog:
+## Setting up Spark Session for Minio
+
+Start a local Minio server using Docker:
```

Comment:
Maybe add a first sentence, that the only major difference is the `spark.sql.catalog.<catalog-name>.s3.endpoint` config.